pkg/lib/cockpit-components-dialog.jsx

```diff
@@ -236,10 +236,11 @@ class DialogFooter extends React.Component {
         );
     }
 }
-DialogFooter.PropTypes = {
+
+DialogFooter.propTypes = {
     cancel_clicked: PropTypes.func,
     cancel_caption: PropTypes.string,
-    actions: PropTypes.array,
+    actions: PropTypes.array.isRequired,
```

Comment:
There is always the cancel button, no?  So "actions" could be optional and default to `[ ]`. But I think it's also okay to requite the caller to specify `[ ]` explicitly.

Reply:
I wasn't aware of the implicit cancel button. Apparently right now, every caller specifies actions, but indeed we could dial this back when we have a case where only a Cancel button is appropriate. For error messages we generally use inline notifications though, so we don't do "error message" popups.